tests/JavascriptTests.cmake

```diff
@@ -212,6 +220,7 @@ function(add_web_client_test case specFile)
   string(REPLACE ";" " " plugins "${plugins}")
 
   set_property(TEST ${testname} PROPERTY ENVIRONMENT
+    "PYTHONPATH=$ENV{PYTHONPATH}"
```

Comment:
1. It's the same thing we do in the [`server tests`](https://github.com/girder/girder/blob/master/tests/PythonTests.cmake#L117).
2. I need to make the `minerva` directory resolvable as a python package.  The only other way is to register it with setuptools.